### PR TITLE
Show student data in rubric when viewing student work

### PR DESCRIPTION
**Pull request by bethanyaconnor**
This PR essentially implements this piece of the [figma](https://www.figma.com/file/WBcs7BY9IC6Nq5rG4V1SWF/AI-in-Teacher-Tools?node-id=2020%3A804&mode=dev) when viewing student work:
Screenshot 2023-08-25 at 9 20 52 AM

What it looks like when a student has attempted the level:
Screenshot 2023-08-25 at 9 12 17 AM

What it looks like when a student hasn't attempted the level (I took a liberty here to show "0 attempts" but no other information):
Screenshot 2023-08-25 at 9 10 20 AM

### CODE REVIEW

File: apps/src/templates/rubrics/rubrics.module.scss [hunk 1]
```diff
@@ -135,7 +175,6 @@ p.aiTokenText {
   margin-bottom: 0;
 }
 
-
```
Comment (bethanyaconnor) [inline apps/src/templates/rubrics/rubrics.module.scss:138]: not sure where all these deleted lines came from but they look like a positive change

File: apps/i18n/common/en_us.json [hunk 2]
```diff
@@ -1586,6 +1587,7 @@
   "notSignedInBody": "You don't need an account to work on this lesson, but if you want to save your work, remember to sign in or create an account before you get started.",
   "notStarted": "Not started",
   "nPoints": "{numPoints, plural, one {1 point} other {# points}}",
+  "numAttempts": "{numAttempts, plural, one {1 attempt} other {# attempts}}",
```
Comment (davidsbailey) [inline apps/i18n/common/en_us.json:1590]: cool! I'd heard this kind of translation was a possibility but never seen it in action before

File: apps/test/unit/templates/rubrics/RubricContainerTest.jsx [hunk 3]
```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import {expect} from '../../../util/reconfiguredChai';
-import {shallow} from 'enzyme';
+import {mount, shallow} from 'enzyme';
```
Comment (davidsbailey) [inline apps/test/unit/templates/rubrics/RubricContainerTest.jsx:3]: this reminds me that we should put our heads together regarding a technical proposal to start using React Testing Library. We tried it out in the curriculum catalog and found no real blockers

**Review by davidsbailey — APPROVED** [review]
Looks great 🤩

File: apps/test/unit/templates/rubrics/RubricContainerTest.jsx (hunk 3 above)
Comment (bethanyaconnor) [reply apps/test/unit/templates/rubrics/RubricContainerTest.jsx:3]: Yes! I read Meg's doc that they wrote before they left and it seems like it was pretty easy to use. I'm slightly regretting not using it for this project now, especially as I'm needing to use `mount` quite a bit anyway

**Pull request merged**